assets/js/blocks/cart-checkout/checkout/inner-blocks/checkout-fields-block/edit.tsx

```diff
@@ -51,20 +42,9 @@ export const Edit = ( {
 		[ 'woocommerce/checkout-billing-address-block', {}, [] ],
 		[ 'woocommerce/checkout-shipping-methods-block', {}, [] ],
 		[ 'woocommerce/checkout-payment-block', {}, [] ],
-		showOrderNotes
-			? [ 'woocommerce/checkout-order-note-block', {}, [] ]
-			: false,
-		showPolicyLinks
-			? [ 'woocommerce/checkout-terms-block', {}, [] ]
-			: false,
-		[
-			'woocommerce/checkout-actions-block',
-			{
-				showReturnToCart,
-				cartPageId,
-			},
-			[],
-		],
```

Comment:
Those should be handled by:
1. Migration code if we're migrating.
2. default attributes if we're inserting fresh.